apps/api/src/app/message-template/usecases/create-message-template/create-message-template.spec.ts

```diff
@@ -57,6 +65,7 @@ describe('Create Message Template', function () {
     expect(result.name).to.eql('test-message-template');
     expect(result.title).to.eql('test');
     expect(result.content).to.eql(content);
-    expect(result.variables).to.eql([]);
+    expect(result.variables?.length).to.eql(1);
+    expect(result.variables?.at(0)?.defaultValue).to.eql('test');
```

Comment:
Shouldn't the message be created with the variable even if the default value is false? And we just shouldn't override the sent message with this information? 

Because this variable is still was used in the template (if that's the usage of the variables array)

And only during trigger we should filter out to not override it with a string. Or maybe I'm missing something @davidsoderberg